node/src/neighborhood/mod.rs

```diff
@@ -1245,13 +1392,55 @@ impl Neighborhood {
         return_route_id
     }
 
+    pub fn find_exit_location<'a>(
+        &'a self,
+        source: &'a PublicKey,
+        minimum_hops: usize,
+        payload_size: usize,
+    ) -> Vec<&'a PublicKey> {
+        let mut minimum_undesirability = i64::MAX;
```

Comment:
Theoretically, you could get rid of this variable, because it's not factually relevant, it is never used here. 

I would definitely tolerate your approach still... so don't worry.

(The same could be thought about the line below.)